### ISSUE TITLE
Unable to install (postbuild and babel not found error message)

### ISSUE THREAD

**Issue by kayode-adechinan** [elierotenberg/fastify-zod#19]
Hello, thanks for the hard work you put on this library. I was able to install the previous version, but lately :

When doing the following on a fresh project:
- npm i fastify-zod

I get some error messages

npm ERR! > fastify-zod@1.0.1 build:babel
npm ERR! > babel src --out-dir build --extensions '.ts' --source-maps
npm ERR! sh: 1: babel: not found

**Comment by Gomah** [comment]
Same error here, @elierotenberg, would you be open to use with a tool like `tsup` instead? I also think it should be "prepublishOnly" instead of "postinstall"?

**Comment by Gomah** [comment]
@elierotenberg I adjust few things on my fork: https://github.com/Gomah/fastify-zod/commits/master , @elierotenberg happy to open a PR if you're keen?

**Comment by elierotenberg** [comment]
You are right. `postinstall` hooks are triggered when the package is installed as a dependency and therefore broke everything.

I've removed the internal `test-openapi-client` package from `devDependencies`, as it is built as part as `npm run build` anyway.

Does that cover your needs?

**Comment by elierotenberg** [comment]
Also please go ahead with the PR, I'm not familiar with `tsup` but this is an opportunity to learn!

**Comment by teddybee** [comment]
1.0.2 has solved the yarn install problem with typescript.
